velox/type/Timestamp.cpp

```diff
@@ -54,7 +54,17 @@ void Timestamp::toGMT(const date::time_zone& zone) {
   date::local_time<std::chrono::seconds> localTime{
       std::chrono::seconds(seconds_)};
   std::chrono::time_point<std::chrono::system_clock, std::chrono::seconds>
-      sysTime = zone.to_sys(localTime, date::choose::latest);
+      sysTime;
+  try {
+    sysTime = zone.to_sys(localTime);
+  } catch (const date::ambiguous_local_time& error) {
```

Comment:
exceptions are expensive; is there a way to avoid throwing and catching an exception here?

Reply:
Unfortunately it's possible without modifying the code of the external date library. But I don't think it would be triggered very often: only 2 hours of the whole year will trigger the exception and the timezone has to be in of the those follow daylight saving.